static/app/views/performance/newTraceDetails/traceDrawer/traceProfilingLink.ts

```diff
@@ -20,6 +24,20 @@ function toDate(value: unknown): Date | null {
   return dateObj;
 }
 
+function getTimeOffsetForDuration(duration: number): number {
+  return duration < 100 ? 100 : duration * 0.1;
```

Comment:
100ms window of time before/after short duration events and 10% of time for longer duration events